### PR TITLE
[Bulk] Validation - Duplicate image alt tag 

### PR DESCRIPTION
**Pull request by vatmicro**
This fixes 9 alt tag issues.

### CODE REVIEW

File: aspnet/web-forms/overview/older-versions-getting-started/continuing-with-ef/handling-concurrency-with-the-entity-framework-in-an-asp-net-web-application.md [hunk 1]
```diff
@@ -130,15 +130,15 @@ Call the new method from the `Updated` event handler that you added earlier. In
 
 Run the *Departments.aspx* page.
 
-[![Image17](handling-concurrency-with-the-entity-framework-in-an-asp-net-web-application/_static/image14.png)](handling-concurrency-with-the-entity-framework-in-an-asp-net-web-application/_static/image13.png)
+[![A screenshot that show the Departments page.](handling-concurrency-with-the-entity-framework-in-an-asp-net-web-application/_static/image14.png)](handling-concurrency-with-the-entity-framework-in-an-asp-net-web-application/_static/image13.png)
```
Comment (jmart1428) [inline aspnet/web-forms/overview/older-versions-getting-started/continuing-with-ef/handling-concurrency-with-the-entity-framework-in-an-asp-net-web-application.md:133]: add an "s" to show

File: aspnet/web-forms/overview/older-versions-getting-started/continuing-with-ef/handling-concurrency-with-the-entity-framework-in-an-asp-net-web-application.md [hunk 2]
```diff
@@ -130,15 +130,15 @@ Call the new method from the `Updated` event handler that you added earlier. In
...
 
 Click **Edit** in a row and change the value in the **Budget** column. (Remember that you can only edit records that you've created for this tutorial, because the existing `School` database records contain some invalid data. The record for the Economics department is a safe one to experiment with.)
 
 [![Image18](handling-concurrency-with-the-entity-framework-in-an-asp-net-web-application/_static/image16.png)](handling-concurrency-with-the-entity-framework-in-an-asp-net-web-application/_static/image15.png)
 
 Open a new browser window and run the page again (copy the URL from the first browser window's address box to the second browser window).
 
-[![Image17](handling-concurrency-with-the-entity-framework-in-an-asp-net-web-application/_static/image18.png)](handling-concurrency-with-the-entity-framework-in-an-asp-net-web-application/_static/image17.png)
+[![A screenshot that show a new browser window.](handling-concurrency-with-the-entity-framework-in-an-asp-net-web-application/_static/image18.png)](handling-concurrency-with-the-entity-framework-in-an-asp-net-web-application/_static/image17.png)
```
Comment (jmart1428) [inline aspnet/web-forms/overview/older-versions-getting-started/continuing-with-ef/handling-concurrency-with-the-entity-framework-in-an-asp-net-web-application.md:141]: add an "s" to show

File: aspnet/web-forms/overview/older-versions-getting-started/continuing-with-ef/handling-concurrency-with-the-entity-framework-in-an-asp-net-web-application.md [hunk 3]
```diff
@@ -227,15 +227,15 @@ Add the following `Page_Init` method, which enables Dynamic Data functionality.
 
 Run the *OfficeAssignments.aspx* page.
 
-[![Image10](handling-concurrency-with-the-entity-framework-in-an-asp-net-web-application/_static/image32.png)](handling-concurrency-with-the-entity-framework-in-an-asp-net-web-application/_static/image31.png)
+[![A screenshot that shows the OfficeAssignments page.](handling-concurrency-with-the-entity-framework-in-an-asp-net-web-application/_static/image32.png)](handling-concurrency-with-the-entity-framework-in-an-asp-net-web-application/_static/image31.png)
```
Comment (jmart1428) [inline aspnet/web-forms/overview/older-versions-getting-started/continuing-with-ef/handling-concurrency-with-the-entity-framework-in-an-asp-net-web-application.md:230]: Let's put a space between Office and Assignments so the screen reader will read it correctly.

File: aspnet/web-forms/overview/older-versions-getting-started/continuing-with-ef/using-the-entity-framework-and-the-objectdatasource-control-part-3-sorting-and-filtering.md [hunk 4]
```diff
@@ -140,7 +140,7 @@ Open the *SchoolRepository.cs* file and specify eager loading for the `Courses`
 
 Run the page. In addition to the sorting and filtering capability that you added earlier, the GridView control now shows nested course details for each department.
 
-[![Image01](using-the-entity-framework-and-the-objectdatasource-control-part-3-sorting-and-filtering/_static/image8.png)](using-the-entity-framework-and-the-objectdatasource-control-part-3-sorting-and-filtering/_static/image7.png)
+[![A screenshot that shows the GridView control displaying nested course details.](using-the-entity-framework-and-the-objectdatasource-control-part-3-sorting-and-filtering/_static/image8.png)](using-the-entity-framework-and-the-objectdatasource-control-part-3-sorting-and-filtering/_static/image7.png)
```
Comment (jmart1428) [inline aspnet/web-forms/overview/older-versions-getting-started/continuing-with-ef/using-the-entity-framework-and-the-objectdatasource-control-part-3-sorting-and-filtering.md:143]: Let's put a space between Grid and View

**Comment by vatmicro** [comment]
Hi @Rick-Anderson ,

**Comment by vatmicro** [comment]
@Rick-Anderson 

**Global effort to fix validation errors**

The Content & Learning team is fixing build validation errors on [docs.microsoft.com](http://docs.microsoft.com/). This effort will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. I’d very much appreciate your timely review and feedback on these changes I’ve suggested. Thank you!

**Pull request merged**